src/layers.js

```diff
@@ -126,97 +78,81 @@ export const vectorLayerCarto = (iso, species, scenario, year, opacity = 1) => {
-//             },
-//             type: 'cartodb'
-//           }
-//         ],
-//         maxzoom: 3,
-//         minzoom: 2,
-//         vectorLayers: [
-//           {
-//             paint: {
-//               'fill-color': [
-//                 'interpolate',
-//                 ['linear'],
-//                 ['get', 'probabilityemca'],
-//                 0,
-//                 transparent,
-//                 0.05,
-//                 transparent,
-//                 0.15,
-//                 transparent,
-//                 0.2,
-//                 transparent,
-//                 0.25,
-//                 transparent,
-//                 0.3,
-//                 transparent,
-//                 0.35,
-//                 transparent,
-//                 0.4,
-//                 transparent,
-//                 0.45,
-//                 transparent,
-//                 0.5,
-//                 COLOR_RAMP[0],
-//                 0.55,
-//                 COLOR_RAMP[1],
-//                 0.6,
-//                 COLOR_RAMP[2],
-//                 0.65,
-//                 COLOR_RAMP[3],
-//                 0.7,
-//                 COLOR_RAMP[4],
-//                 0.75,
-//                 COLOR_RAMP[5],
-//                 0.8,
-//                 COLOR_RAMP[6],
-//                 0.85,
-//                 COLOR_RAMP[7],
-//                 0.9,
-//                 COLOR_RAMP[8],
-//                 0.95,
-//                 COLOR_RAMP[9],
-//                 1,
-//                 COLOR_RAMP[10]
-//               ],
-//               'fill-opacity': opacity
-//             },
-//             'source-layer': 'layer0',
-//             type: 'fill'
-//           }
-//         ]
-//       },
-//       params_config: [],
-//       sql_config: [
-//         {
-//           key: 'where',
-//           key_params: [
-//             {
-//               key: 'iso3',
-//               required: true
-//             }
-//           ]
-//         }
-//       ]
-//     },
-//     provider: 'cartodb'
-//   };
-// };
+export const currentDistributionCartoLayer = (iso, species, opacity = 1) => {
+  const SCENARIO = 'current';
+
+  return {
+    id: `${iso}${species}${SCENARIO}`,
+    name: 'Current distribution carto layer',
+    sqlParams: {
+      where: {
+        iso3: iso
+      },
+      where2: {
+        species
+      }
+    },
+    layerConfig: {
+      account: 'simbiotica',
+      body: {
+        layers: [
+          {
+            options: {
+              sql: `WITH a AS (SELECT cartodb_id, the_geom_webmercator, uuid, iso3 FROM all_geometry {{where}}) SELECT a.the_geom_webmercator, a.cartodb_id, b.uuid, b.timeinterval, b.species, b.scenario, b.probabilityemca FROM ${iso.toLowerCase()}_zonal_spp_uuid as b INNER JOIN a ON b.uuid = a.uuid {{where2}}`
```

Comment:
I'm probably missing something. Shouldn't we be filtering out by year === 1995?

Reply:
We're not passing a scenario or year value, I hard-coded scenario value for current distributions:
![image](https://user-images.githubusercontent.com/15097138/68771013-1851fa80-061f-11ea-83f1-74037db84f26.png)
and then I set this as a default for a scenario param, so filtering by year it's not needed here I think ;)
![image](https://user-images.githubusercontent.com/15097138/68770908-eb9de300-061e-11ea-92ca-e8081b3a4d54.png)